vs_scanwatch/Service1.cs

```diff
@@ -0,0 +1,325 @@
+    {        
+        public int error;
+        public FileStream[] fileStreams;    //TODO dispose of array resources on each run
+                                            //fileStreams need to be done through an instance of a separate class to ensure that they get properly disposed once a successful copy has been confirmed
+    }
+    static class FolderPath //This class exists just to pass the App.config settings from the OnStart to the OnElapsedTime method
+    {
+        public static string rootDirectoryPath; //this needs to be a valid path to an existing folder, if path is not valid/folder does not exist default value is used
+                                                //default value is "C:\Scan\" and if that folder doesn't exist "C:\Scan\" will be created
+        public static string outputFolderPath;  //this needs to be formated as yyyyMMdd and needs to be a valid date to be accepted
+                                                //default value is current date in Belgrade formated as yyyyMMdd
+    }
+    public partial class Service1 : ServiceBase
+    {
+        Timer timer = new Timer();
+        public Service1()
+        {
+            InitializeComponent();
+        }
+        
+        protected override void OnStart(string[] args)
+        {            
+            string overrideFolderPath = ConfigurationManager.AppSettings["overrideFolderPath"];    //This can be loaded, but has a default value
+            if ("false" == overrideFolderPath || "FALSE" == overrideFolderPath || "False" == overrideFolderPath)
+            {
+                overrideFolderPath = default(string);
+            }
+
+            string overrideDate = ConfigurationManager.AppSettings["overrideDate"];
+            FolderPath.rootDirectoryPath = SetRootDirectory(overrideFolderPath);
+
+            string debug = ConfigurationManager.AppSettings["debug"];
+
+            string logName = "\\log\\LogFile_Debug_" + TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow, TimeZoneInfo.FindSystemTimeZoneById("Central Europe Standard Time")).ToString("yyyy.MM.dd") + ".log";
+
+            if ("true" != debug && "TRUE" != debug && "True" != debug)
+            {
+                logName = "\\log\\LogFile_" + TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow, TimeZoneInfo.FindSystemTimeZoneById("Central Europe Standard Time")).ToString("yyyy.MM.dd") + ".log";
+            }
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Debug()
+                .WriteTo.File(FolderPath.rootDirectoryPath + "\\log\\LogFile_Error.txt", restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Warning)
+                .WriteTo.File(FolderPath.rootDirectoryPath + logName, restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Debug)
+                .CreateLogger();
+
+            if ("true" != debug && "TRUE" != debug && "True" != debug)
```

Comment:
instead of using this logic, you should use built in try parse methods for boolean, e.g.

```bool.Parse(input)```